lalrpop-test/src/error_issue_278.lalrpop

```diff
@@ -0,0 +1,11 @@
+use lalrpop_util::ParseError;
+
+grammar;
+
+extern {
+	type Error = &'static str;
+}
+
+pub Value = <Int> "abc";
+
+Int: u64 = r"\d+" =>? Err(ParseError::User{ error: "Pretend there was an error" });
```

Comment:
Might want to set your editor to add new lines at the end of files - doesn't play nicely with diffs.

Reply:
Fixed, thanks. I agree that it is annoying without a newline at the end of the file. The problem was that I didn't use an editor for that file.